config/crd/kustomizeconfig.yaml

```diff
@@ -12,6 +12,3 @@ namespace:
   group: apiextensions.k8s.io
   path: spec/conversion/webhookClientConfig/service/namespace
   create: false
-
-varReference:
-- path: metadata/annotations
```

Comment:
Why is this removed? It is still in the kubebuilder skaffolding AFAICS:
 * https://github.com/kubernetes-sigs/kubebuilder/blob/18df53806e97f5ed41161edeb7607bf0f04b1d07/pkg/plugins/common/kustomize/v2/scaffolds/internal/templates/config/crd/kustomizeconfig.go#L62-L64